cmake/AddChmTarget.cmake

```diff
@@ -42,7 +43,6 @@ function (add_chm_target docname lang entities figures dtd_files)
         WORKING_DIRECTORY "${BUILD_DIR}"
         DEPENDS "${BUILD_DIR}/htmlhelp.hhp"
                 "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-fig-trigger"
-                "${CMAKE_CURRENT_BINARY_DIR}/${fmt}-xslticon-trigger"
```

Comment:
Why do you drop this dependency ? Is htmlhelp.hpp not dependent on xslt icons but it is on figures ?